### PR TITLE
Various tweaks for the kernel job

### PR DESCRIPTION
**Pull request by idryomov**

### CODE REVIEW

**Comment by idryomov** [comment]
Jenkins check failure looks unrelated.

File: kernel/build/build_deb [hunk 1]
```diff
@@ -1,11 +1,6 @@
 #!/bin/bash
 set -ex
 
-# Only do actual work when we are a DEB distro
```
Comment (alfredodeza) [inline kernel/build/build_deb]: why remove this? it will execute if this is running on an RPM box

File: kernel/build/build_rpm [hunk 2]
```diff
@@ -1,11 +1,6 @@
 #!/bin/bash
 set -ex
 
-# Only do actual work when we are an RPM distro
```
Comment (alfredodeza) [inline kernel/build/build_rpm]: same here

Comment (idryomov) [inline kernel/build/build_rpm]: @alfredodeza Oh, should have been more clear - because we already have validate_{deb,rpm} in the definition.

**Comment by alfredodeza** [comment]
jenkins test this please

**Pull request merged**